README.md

```diff
@@ -1,6 +1,8 @@
-# neostarfall [![License](https://img.shields.io/github/license/neostarfall/neostarfall?color=red)](LICENSE) [![Discord](https://img.shields.io/discord/1350996201713045594?label=Discord&logo=discord&logoColor=ffffff&labelColor=7289DA&color=2c2f33)](https://discord.gg/aSXXa4urpm)
-
-> Starfall, but not by groomers :v)
+<div align="center">
+    <img src="./branding/neostarfall_logo_512x512.png" width="256" height="256" />
```

Comment:
I think this should be a clickable link, either to the helper or back to the repo.

Usually for big images it's good to do in case of clicking it on accident and seeing the image which chances are nobody wants to do

Reply:
I'll just do this in post